asf_lifetime_cost_model/getters/data_getters.py

```diff
@@ -0,0 +1,394 @@
+    list_levies = [
+        levies.RO.from_dataframe(
+            levies_data_getters.process_data_RO(fileobject),
+            denominator=supply_elec,
+            price_cap=price_cap_period,
+        ),
+        levies.AAHEDC.from_dataframe(
+            levies_data_getters.process_data_AAHEDC(fileobject),
+            denominator=supply_elec,
+            price_cap=price_cap_period,
+        ),
+        levies.GGL.from_dataframe(
+            levies_data_getters.process_data_GGL(fileobject),
+            denominator=customers_gas,
+            price_cap=price_cap_period,
+        ),
+        levies.WHD.from_dataframe(
+            levies_data_getters.process_data_WHD(fileobject),
+            customers_gas=customers_gas,
+            customers_elec=customers_elec,
+            price_cap=price_cap_period,
+        ),
+        levies.ECO4.from_dataframe(
+            levies_data_getters.process_data_ECO(fileobject), price_cap=price_cap_period
+        ),  # Split ECO
+        levies.GBIS.from_dataframe(
+            levies_data_getters.process_data_ECO(fileobject), price_cap=price_cap_period
+        ),  # Split ECO
+        levies.FIT.from_dataframe(
+            levies_data_getters.process_data_FIT(fileobject),
+            scaling_factor=fit_scaling_factor,
+            price_cap=price_cap_period,
+        ),
+        levies.NCC.from_dataframe(
+            levies_data_getters.process_data_NCC(fileobject),
+            scaling_factor=ncc_scaling_factor,
+            price_cap=price_cap_period,
+        ),
+    ]
+    fileobject.close()
+
+    levy_collection = levies.LevyCollection(
+        "Policy Costs", "pc", list_levies, denominator_values
+    )
+
+    return levy_collection
+
+
+def get_rebalanced_levies(
+    scenario_name: str, price_cap_period: str = "LATEST"
+) -> LevyCollection:
+    """
+    Create a LevyCollection containing Levy objects that have been rebalanced according to the scenario provided.
+
+    Args:
+        scenario_name (str): Rebalancing scenario, valid arguments are "Remove all", "Rebalance RO and FiT to gas",\
+            "Remove from electricity", "Rebalance electricity unit cost levies to gas", "Rebalance all to gas".
+        price_cap_period (str, optional): Date of interest in YYYY-MM-DD format. \
+            "LATEST" is also valid to get the most recently available price cap.. Defaults to "LATEST".
+
+    Raises:
+        KeyError: If unrecognised scenario is provided.
+
+    Returns:
+        LevyCollection: LevyCollection object containing rebalanced Levy objects, \
+            each representing a levy present in the policy costs component of the price cap period provided.
+    """
+
+    """
+    Rebalance levies according to supplied denominators (consumption and customer charging base). \
+        Note: This is used for internal consistency in rebalancing as different charging base numbers are \
+            used across different levies to determine levy rates.
+    """
```

Comment:
i find it more readable like this

```suggestion
    # Rebalance levies according to supplied denominators (consumption and customer charging base). \
    # Note: This is used for internal consistency in rebalancing as different charging base numbers are \
    # used across different levies to determine levy rates.
    """
```